packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2584,10 +2615,20 @@ class _InputDecoratorState extends State<InputDecorator> with TickerProviderStat
               : const EdgeInsetsDirectional.fromSTEB(12.0, 24.0, 12.0, 16.0));
     }
 
+    double inputGap = 0.0;
```

Comment:
Nit: You could probably refactor a bit to make this `final`, but maybe it would be harder to read that way 🤷

Reply:
I kept it that way because the result of autoformat when using ternary or switch was very bad.